modules/core/src/v2/wallet.ts

```diff
@@ -1359,8 +1366,27 @@ export class Wallet {
     // get keychains for address verification
     const keychains = await Promise.all(this._wallet.keys.map((k) => this.baseCoin.keychains().get({ id: k, reqId })));
     const rootAddress = _.get(this._wallet, 'receiveAddress.address');
+    const addressDerivationKeypair = _.get(keychains[KeyIndices.USER], 'addressDerivationKeypair');
```

Comment:
@joey-bai  this will always exist, right?

Reply:
right

Reply:
this may not exist since we only create it if `this.baseCoin.supportsDerivationKeypair() === true`.